src/route53.ts

```diff
@@ -67,6 +67,7 @@ export const needsUpdateRecord = async (
   const { ResourceRecordSets } = await route53
     .listResourceRecordSets({
       HostedZoneId: hostedZoneId,
+      StartRecordName: domainName,
```

Comment:
I guess you can remove the TODO 😁

Reply:
Ah yes thanks I saw it and din't even understand it and now I do ^^